helm_deploy/cla-backend/templates/deployment.yaml

```diff
@@ -94,7 +94,7 @@ spec:
         - name: {{ include "cla-backend.fullname" . }}-db
           securityContext:
             {{- toYaml .Values.securityContext | nindent 12 }}
-          image: "bitnami/postgresql:14-debian-10"
+          image: "bitnamilegacy/postgresql:14-debian-12"
```

Comment:
@willc-work should we use this version in `config.yml` where we are using `postgres:14.7-bullseye` for consistency?

I guess we don't want to change `postgres:11.6-alpine` in `docker-compose.yml`?

Reply:
I will monitor the build time to see if there is much difference as it isnt using an alpine version.